tcpdf.php

```diff
@@ -7558,6 +7558,8 @@ public function Output($name='doc.pdf', $dest='I') {
 		}
 		$dest = strtoupper($dest);
 		if ($dest[0] != 'F') {
+			$name = explode("/", $name);
```

Comment:
Why explode() the file name rather than pathinfo()? If explode is better, why explode on "/" rather than `DIRECTORY_SEPARATOR`?

Reply:
I agree, that this should be `pathinfo` for x-platform compat